pkg/actions/extract_index.go

```diff
@@ -39,10 +39,10 @@ func ExtractIndexDB(image string, containerEngine string) error {
 	}
 
 	// Extract
-	command = exec.Command(containerEngine, "cp", fmt.Sprintf("%s:/database/index.db", catalogIndex), "./output/")
+	command = exec.Command(containerEngine, "cp", fmt.Sprintf("%s:/database/one.db", catalogIndex), "./output/")
```

Comment:
index here should be fine

Reply:
Will be removing this action since it is not being used opcap anymore.